src/sidebar/test/search-client-test.js

```diff
@@ -10,142 +10,189 @@ function delay(ms) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-describe('SearchClient', () => {
-  const RESULTS = [
-    { id: 'one' },
-    { id: 'two' },
-    { id: 'three' },
-    { id: 'four' },
-  ];
+const RESULTS = [
+  // nb. `created` and `updated` dates are in opposite order and different months
+  // to make it easy to check in tests that correct field was used for searching/sorting.
+  { id: 'one', created: '2020-01-01', updated: '2020-02-04' },
+  { id: 'two', created: '2020-01-02', updated: '2020-02-03' },
+  { id: 'three', created: '2020-01-03', updated: '2020-02-02' },
+  { id: 'four', created: '2020-01-04', updated: '2020-02-01' },
+];
+
+/**
+ * Fake implementation of the `/api/search` API
+ */
+async function executeSearch(params) {
```

Comment:
The sophistication of this fake makes me a little twitchy, but in reading through the set of tests, I don't have a better idea, either.